Dockerfile

```diff
@@ -2,6 +2,10 @@ FROM quay.io/opendatahub/text-generation-inference:fast-283ec87
 
 USER root
 
+# Add grpc-ecosystem health probe
+# This is from https://github.ibm.com/ai-foundation/caikit-pt/blob/main/Dockerfile
```

Comment:
nit: This is probably leaking internal information so we might want to remove the link here.